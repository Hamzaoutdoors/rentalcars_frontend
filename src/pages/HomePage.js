import React from 'react';
import styled from 'styled-components';
import Slider from '../components/Slider';
// import Categories from '../components/Categories';
// import Products from '../components/Products';
// import Newsletter from '../components/Newsletter';
// import Footer from '../components/Footer';
// import { desktop, mobile } from '../responsive';

const Container = styled.div`
  display: flex;
  flex-direction: column;
  width: 100vw;
  height: 100vh;
  color: black;
  justify-content: center;
  align-items: center;
  overflow-x: hidden;
`;

const Title = styled.h1`
  font-size: 2.2em;
  text-align: center;
  font-weight: bold;
`;

const SubTitle = styled.h2`
  font-size: 0.8em;
  text-align: center;
  letter-spacing: 1px;
  padding-bottom: 20px;
  font-weight: bold;
  color: gray;
  border-bottom: 2px dotted #D3D3D3;

`;

const HomePage = () => (
  <Container>
    <Title> LATEST MODELS</Title>
    <SubTitle>Please select your car</SubTitle>
    <Slider />
  </Container>
);

export default HomePage;
