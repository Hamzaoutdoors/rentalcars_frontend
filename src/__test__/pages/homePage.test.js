import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import HomePage from '../../pages/HomePage';
import store from '../../redux/configureStore';

it('render the Home page correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <HomePage />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
