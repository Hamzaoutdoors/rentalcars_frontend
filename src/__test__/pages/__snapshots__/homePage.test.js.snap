// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Home page correctly 1`] = `
<div
  className="sc-bdvvtL gFuYZJ"
>
  <a
    className="sc-gsDKAQ lbFdcj"
    href="/cars"
    onClick={[Function]}
  >
    Start your journey
  </a>
</div>
`;
